### ISSUE TITLE
if no loot was collected for chosen resolution, breaks everything

### ISSUE THREAD

**Issue by samuelhackwill** [samuelhackwill/le-tiret-du-six#81]
expected behaviour : players can choose any conclusion at the end of the beach sequence

what's going on : if players have no loot, we get an error in the console 

```
TypeError : instance.data.obj.modifiers.find(...) is undefined
```

steps to replicate : just go to the end of the beach sequence

**Comment by samuelhackwill** [comment]
solved in  dd48185

**Comment by samuelhackwill** [comment]
solved in 20b0a4c
